requirements.txt

```diff
@@ -2,25 +2,22 @@ asgiref==3.2.10
 certifi==2020.6.20
 cffi==1.14.2
 chardet==3.0.4
-cryptography==3.0
+cryptography==3.3.2
 defusedxml==0.7.0rc1
-Django==3.1
+Django==3.1.8
```

Comment:
Django updated to 3.1.8,
some of the packages were removed, as they were not needed